data/Italy/House/sources/instructions.json

```diff
@@ -11,6 +11,23 @@
       "type": "membership"
     },
     {
+      "file": "morph/camera.csv",
+      "create": {
+        "type": "morph",
+        "scraper": "andylolz/italy-camera",
+        "query": "SELECT * FROM data"
+      },
+      "source": "https://morph.io/andylolz/italy-camera",
```

Comment:
Would you mind setting this to the 'end' source — i.e. the site that gets scraped, rather than scraper itself? This isn't really documented yet, or even _used_ yet, but I hope to do https://github.com/everypolitician/everypolitician/issues/127 pretty soon, which will actually display all these `source` fields, and it would be better if they were where the data came from, rather than the mechanism used to get them (if that makes sense?)